src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { tooltipStyle, tooltipChildrenWrapperStyle } from './Tooltip.style';
-import { uuid } from 'uuidv4';
+import { v4 } from 'uuid';
```

Comment:
why you use `uuid` here? is not part of the packages - maybe is one of the libraries being called by another library that maybe in the future will break.

Reply:
This was your change actually, I just changed it from uuid() to v4(), because the first one is deprecated :)